navigation_and_routing/lib/src/routing/delegate.dart

```
@@ -0,0 +1,52 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'parsed_route.dart';
+import 'route_state.dart';
+
+class SimpleRouterDelegate extends RouterDelegate<ParsedRoute>
+    with ChangeNotifier, PopNavigatorRouterDelegateMixin<ParsedRoute> {
+  final RouteState routeState;
+  final WidgetBuilder builder;
+
+  @override
+  final GlobalKey<NavigatorState> navigatorKey;
+
+  SimpleRouterDelegate({
+    required this.routeState,
+    required this.builder,
+    required this.navigatorKey,
+    // ignore: prefer_initializing_formals
```

Is this actually ignoring an error here?